### ISSUE TITLE
[2] add player name and turf inked to battle results page's team table

### ISSUE THREAD

**Issue by hymm** [fetus-hina/stat.ink#265]
add player name and turf inked to battle results for team members.

![image](https://user-images.githubusercontent.com/2180432/29501398-2f48a5f0-85dd-11e7-9fec-1db4bb7fa47c.png)

**Comment by fetus-hina** [comment]
Turf inked: I will add a "point ↔ inked" switcher

Players' name:  I'm thinking how to handle it. I believe it shouldn't be public. Because stat.ink is just stats site and not for pillory. For example, the "blackout" option is a part of such efforts.

**Comment by fetus-hina** [comment]
mmm... Is meaning "display turf inked in **Ranked Battle**"?

**Comment by hymm** [comment]
Is it possible to just make names available to the owner of a battle?

Yes, I did did mean turf inked for ranked.

**Comment by hymm** [comment]
I see there being 4 options for names:

1.  Do not ever show the user:  No names means no possibility of identifying other players.

2.  Only allow owner of battle to see names for league and private battles:  This would allow the uploader to see how their friends stats.

3.  Make names public for for league and private battles:  This would allow your friends to see their stats too.

4.  Make names public for everything:  Splatnet2 already allows you to share battle information with names included.  Including this information in stat.ink is only a little more harmful.

My opinion is that 3 is probably a good compromise.

**Comment by fetus-hina** [comment]
Added turf inked on Ranked Battle in commit 4c194156cbfd1d35de1bb29421f095b5d31932e6

**Comment by fetus-hina** [comment]
I implemented this feature and deployed it.

- New configuration for blacking out for details list was added.
  - The default value of this configuration is like your option 3.
- If logged in as a battle's owner, all players' name will be shown. (This behavior has been implemented in commit 9862985127f2b96c8c4ce0cad9be52bd668b7d13 and deployed 2 days ago)
- If not logged in, the new configuration will apply (new behavior). Options are:
  - Always opened all players' name
  - Opened if private battle
  - Opened if private battle or league(4)'s team member
  - Always masked
